src/main/resources/i18n/messages_cs.properties

```diff
@@ -758,4 +758,7 @@ chat.userContext.report = Nahlásit hráče
 userInfo.ratingHistory.lastYear = Poslední rok
 userInfo.ratingHistory.lastMonth = Poslední měsíc
 game.generateMap.less = Méně
-game.generateMap.more = Více
\ No newline at end of file
+game.generateMap.more = Více
+settings.data.mirrorURLs=Download Mirrors
+settings.data.mirrorURLs.description=Add any mirrors here that you wish to use for downloading game patches instead of downloading from the main FAF server. This can help prevent extremely long download times.
+settings.data.mirrorURLPrompt=https://faf-mirror.example.com/
```

Comment:
These need to be removed from all but the message.properties files

Reply:
Weird I thought I did that already… will take another look.

Reply:
Ah you probably did I wiped some changes with the rebase but thought I fixed them all. So I will go ahead and fix this

Reply:
Since I caused it